share/spice/launch_library/launch_library.js

```diff
@@ -0,0 +1,72 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_launch_library = function(api_result){
+
+        // If the response failed on the server, don't render the Instant Answer at all
+        if (!api_result || api_result.status == "fail" || api_result.total === undefined) {
+            return Spice.failed('launch_library');
+        }
+
+        if (api_result.total > 0) {
+            DDG.require("moment.js", function() {
+                Spice.add({
+                    id: "launch_library",
+
+                    name: "Answer",
+                    data: api_result.launches,
+                    meta: {
+                        itemType: (api_result.number === 1 ? "Upcoming Launch" : "Upcoming Launches"),
+                        sourceName: "LaunchLibrary.net",
```

Comment:
Could we make this just "LaunchLibrary"? We don't usually append the last part. For example, https://duckduckgo.com/?q=ray+bradbury+books&ia=products doesn't say "Amazon.com" but instead just "Amazon". :)